libsolidity/lsp/LanguageServer.cpp

```diff
@@ -210,6 +211,40 @@ void LanguageServer::changeConfiguration(Json::Value const& _settings)
 		if (typeFailureCount)
 			m_client.trace("Invalid JSON configuration passed. \"include-paths\" must be an array of strings.");
 	}
+
+	Json::Value jsonRemappings = _settings["remappings"];
+
+	if (jsonRemappings)
+	{
+		bool typeFailures = false;
+		if (jsonRemappings.isArray())
+		{
+			vector<frontend::ImportRemapper::Remapping> remappings;
+			for (Json::Value const& jsonPath: jsonRemappings)
+			{
+				if (
+					jsonPath.isObject() &&
+					jsonPath.get("context", "").isString() &&
+					jsonPath.get("prefix", "").isString() &&
+					jsonPath.get("target", "").isString()
+				)
```

Comment:
This is missing a validation that `target` is not empty. solc won't accept remappings with an empty target on the CLI.